File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/HeaderedPanel/Features/HeaderedPanelTextPropertyPage.vb
Namespace CompanionKit.Controls.HeaderedPanel.Features

	Public Class HeaderedPanelTextPropertyPage

        Public Sub New()
            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()
        End Sub

        ''' <summary>
        ''' Handles Click event for Button control
        ''' </summary>
        Private Sub mobjButton_Click(ByVal sender As Object, ByVal e As EventArgs) Handles mobjButton.Click
            ' Set header text for HeaderedPanel control
            Me.mobjHeaderedPanel.Text = Me.mobjTextBox.Text
        End Sub

	End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/HeaderedPanel/Features/HeaderedPanelTextPropertyPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.HeaderedPanel.Features
{
    public partial class HeaderedPanelTextPropertyPage : UserControl
    {
        public HeaderedPanelTextPropertyPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Click event for Button control
        /// </summary>
        private void mobjButton_Click(object sender, EventArgs e)
        {
            // Set header text for HeaderedPanel control
            mobjHeaderedPanel.Text = mobjTextBox.Text;
        }
    }
}